examples/gno.land/r/demo/groups/misc.gno

```diff
@@ -0,0 +1,121 @@
+package groups
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/r/demo/users"
+)
+
+//----------------------------------------
+// private utility methods
+// XXX ensure these cannot be called from public.
+
+func getGroup(gid GroupID) *Group {
+	gidkey := groupIDKey(gid)
+	group_, exists := gGroups.Get(gidkey)
+	if !exists {
+		return nil
+	}
+	group := group_.(*Group)
+	return group
+}
+
+func incGetGroupID() GroupID {
+	gGroupsCtr++
+	return GroupID(gGroupsCtr)
+}
+
+func padLeft(str string, length int) string {
+	if len(str) >= length {
+		return str
+	} else {
+		return strings.Repeat(" ", length-len(str)) + str
+	}
+}
+
+func padZero(u64 uint64, length int) string {
+	str := strconv.Itoa(int(u64))
+	if len(str) >= length {
+		return str
+	} else {
+		return strings.Repeat("0", length-len(str)) + str
+	}
+}
```

Comment:
maybe:
```go
func padZero(u64 uint64, length int) string {
	return fmt.Sprintf("%0*d", length, u64)
}
```
It saves you from typecasting `uint64` to `int` in L39 and also makes the code efficient and much easier to read.

Reply:
same error as above if we use fmt